INSTALL.rst

```diff
@@ -1,123 +1,104 @@
 Installing scikit-image
 =======================
 
-We are assuming that you have default Python environment already configured on
-your computer and you intend to install ``scikit-image`` inside of it. If you
-want to create and work with Python virtual environments, please follow the
-instructions on `venv`_ and `virtual environments`_.
-
-There are two ways you can install ``scikit-image`` on your preferred Python
-environment.
-
-1. Standard Installation
-2. Development Installation
-
-1. Standard Installation:
--------------------------
-
-``scikit-image`` comes pre-installed with several Python distributions,
-including `Anaconda <https://www.anaconda.com/download/>`_,
-`Enthought Canopy <https://www.enthought.com/product/canopy/>`_,
-`Python(x,y) <https://python-xy.github.io/>`_ and
-`WinPython <https://winpython.github.io/>`_.
-
-On all major operating systems, install it via shell/command prompt::
-
+We highlight four different ways to install and use ``scikit-image`` for
```

Comment:
What if this was _We highlight four different ways to install and use ``scikit-image``_, since we do not support Python 2 anymore (or other programming languages)? :)